frontend/src/reducers/customizationReducer.test.tsx

```diff
@@ -0,0 +1,38 @@
+import { CustomizationType } from '../actions/customizationActions'
+import reducer, {initialState} from './customizationReducer'
+
+describe('customizationReducer', () => {
+  it('should handle SET_USER_CUSTOMIZATION', () => {
+    expect(
+      reducer(initialState, {
```

Comment:
let't define initial state in test to make sure that it is different from expected.